src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js

```diff
@@ -75,7 +76,11 @@ define([
 
             templates: {
                 skeleton: skeletonTemplate,
-                uploadUrl: '/admin/api/media?collection=<%= id %>&locale=<%= locale %><% if (!!types) {%>&types=<%= types %><% } %>'
+                url: [
+                    '<%= url %>?locale=<%= locale %>',
+                    '<% if (!!types) {%>&types=<%= types %><% } %>',
+                    '<% _.each(params, function(value, key) {%>&key=<%= value %><% }) %>'
```

Comment:
This line actually creates the key `key`, which makes the upload in the overlay stop working.